File: util/src/main/java/at/shockbytes/util/adapter/ItemTouchHelperAdapter.java
package at.shockbytes.util.adapter;

/**
 * @author Martin Macheiner
 * Date: 09.09.2015.
 */
public interface ItemTouchHelperAdapter {

    boolean onItemMove(int from, int to);

    void onItemMoveFinished();

    void onItemDismiss(int position);

}

File: util/src/main/java/at/shockbytes/util/adapter/ItemTouchHelperAdapter.kt
package at.shockbytes.util.adapter

/**
 * Author:  Martin Macheiner
 * Date:    09.09.2015
 */
interface ItemTouchHelperAdapter {

    fun onItemMove(from: Int, to: Int): Boolean

    fun onItemMoveFinished()

    fun onItemDismiss(position: Int)
}
